sql/moz-fx-data-shared-prod/subscription_platform/apple_subscriptions/view.sql

```diff
@@ -0,0 +1,226 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.subscription_platform.apple_subscriptions`
+AS
+WITH apple_iap_events AS (
+  SELECT
+    document_id,
+    -- WARNING: field order from here must exactly match guardian_apple_events_v1
+    CAST(NULL AS STRING) AS legacy_subscription_id,
+    `timestamp` AS event_timestamp,
+    mozfun.iap.parse_apple_event(`data`).*,
+  FROM
+    -- `moz-fx-fxa-prod-0712.firestore_export.iap_app_store_purchases_raw_changelog`
+    -- Table does not yet exist in prod, so use a placeholder
+    UNNEST(ARRAY<STRUCT<document_id STRING, `timestamp` TIMESTAMP, `data` STRING>>[])
```

Comment:
Can they somehow get the table created in prod before the migration?  Otherwise we're stuck waiting to have to reactively modify and deploy this PR whenever the migration ends up happening.

Reply:
302 -> https://mozilla-hub.atlassian.net/browse/FXA-5890?focusedCommentId=625866